rules/windows/process_creation/win_susp_copy_lateral_movement.yml

```diff
@@ -1,31 +1,49 @@
 title: Copy from Admin Share
 id: 855bc8b5-2ae8-402e-a9ed-b889e6df1900
 status: experimental
-description: Detects a suspicious copy command from a remote C$ or ADMIN$ share
+description: Detects a suspicious copy command to or from an Admin share
 references:
     - https://twitter.com/SBousseaden/status/1211636381086339073
-author: Florian Roth
+    - https://drive.google.com/file/d/1lKya3_mLnR3UQuCoiYruO3qgu052_iS_/view
+author: Florian Roth, Teymur Kheirkhabarov '@HeirhabarovT', Zach '@svch0st'
```

Comment:
could you please put the quotes only once, on both ends of the author field? that's a common style, I am not sure if it will break something, but still: 

```
 grep -irh 'author: ' ./ | grep -e '"' -e "'" | sort | uniq
author: "@BarryShooshooga"
author: "@SBousseaden, Florian Roth"
author: "Dimitrios Slamaris, @atc_project (fix)"
author: "Lionel PRAT, Christophe BROCAS, @atc_project (improvements)"
author: '@2xxeformyshirt (Security Risk Advisors) - rule; Teymur Kheirkhabarov (idea)'
author: '@41thexplorer, Microsoft Defender ATP'
author: '@ROxPinTeddy'
author: '@SBousseaden (detection), Thomas Patzke (rule)'
author: '@caliskanfurkan_'
author: '@neu5ron'
author: '@neu5ron, Florian Roth'
author: '@neu5ron, SOC Prime'
author: '@neu5ron, Teymur Kheirkhabarov, oscd.community'
author: 'Furkan Caliskan (@caliskanfurkan_)'
author: 'Josh Brower @DefensiveDepth'
author: 'Nate Guagenti (@neu5ron), Open Threat Research (OTR)'
author: 'SOC Prime, Adam Swan'
author: 'Samir Bousseaden, @neu5rn'
author: 'Samir Bousseaden, @neu5ron'
```